virttest/utils_test/libvirt.py

```diff
@@ -689,29 +689,7 @@ def verify_virsh_console(session, user, passwd, timeout=10, debug=False):
     log = ""
     console_cmd = "cat /proc/cpuinfo"
     try:
-        while True:
-            match, text = session.read_until_last_line_matches(
-                [r"[E|e]scape character is", r"login:",
-                 r"[P|p]assword:", session.prompt],
-                timeout, internal_timeout=1)
-
-            if match == 0:
-                if debug:
-                    LOG.debug("Got '^]', sending '\\n'")
-                session.sendline()
-            elif match == 1:
-                if debug:
-                    LOG.debug("Got 'login:', sending '%s'", user)
-                session.sendline(user)
-            elif match == 2:
-                if debug:
-                    LOG.debug("Got 'Password:', sending '%s'", passwd)
-                session.sendline(passwd)
-            elif match == 3:
-                if debug:
-                    LOG.debug("Got Shell prompt -- logged in")
-                break
-
+        virsh_console_login(session, user, passwd, timeout, debug=debug)
```

Comment:
Do I miss something here? I don't notice code changes except that bringing previous code into one independent method virsh_console_login().

Reply:
Previous function always close console session and just return True or False. If so , We will be unable to execute some cmd with console session with previous function.